extensions/1bookmark/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "1bookmark",
+  "title": "1bookmark",
+  "description": "One Bookmark Solution for Teams and Personal Use.",
+  "type": "module",
+  "icon": "1bookmark-512x512.png",
+  "access": "public",
+  "author": "onebookmark",
+  "contributors": [
+    "b6pzeusbc54tvhw5jgpyw8pwz2x6gs",
+    "lee_juwon",
+    "naru200"
+  ],
+  "categories": [
+    "Productivity",
+    "Applications"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "add-bookmark",
+      "title": "Add Bookmark",
+      "description": "Add a new bookmark",
+      "mode": "view"
+    },
+    {
+      "name": "search-bookmarks",
+      "title": "Search Bookmarks",
+      "description": "Search bookmark and open it",
+      "mode": "view"
+    }
+  ],
+  "preferences": [],
+  "dependencies": {
+    "@raycast/api": "^1.89.1",
+    "@raycast/utils": "^1.18.1",
+    "@tanstack/query-async-storage-persister": "^5.54.1",
+    "@tanstack/react-query": "^5.55.0",
+    "@tanstack/react-query-persist-client": "^5.55.0",
+    "@trpc/client": "^11.0.0-rc.604",
+    "@trpc/react-query": "^11.0.0-rc.604",
+    "@trpc/server": "^11.0.0-rc.604",
+    "axios": "^1.7.7",
+    "jotai": "^2.10.1",
+    "minisearch": "^7.1.1",
+    "node-fetch": "^3.3.2",
+    "radash": "^12.1.0",
+    "superjson": "^2.2.1",
+    "ulid": "^2.3.0"
+  },
+  "devDependencies": {
+    "@raycast/eslint-config": "^1.0.11",
+    "@types/node": "20.8.10",
+    "@types/react": "18.3.3",
+    "eslint": "^8.57.0",
+    "eslint-plugin-only-warn": "^1.1.0",
+    "eslint-plugin-react-hooks": "^5.1.0",
+    "prettier": "^3.3.3",
+    "typescript": "^5.4.5"
+  },
+  "scripts": {
+    "build": "echo \"ray 커맨드가 Turbo 환경을 지원하지 않아 GitHub Actions에서만 빌드하고 있어요\"",
```

Comment:
What is this?

Reply:
The private repository of this extension manages projects as a monorepo, and this extension is one of the monorepo configurations. 

Since Raycast build requires a few more steps, we are building it directly using `ray` in CI, and the echo command is a measure to prevent the build command from being executed accidentally. If you need to change this to the ray build command, it can be changed.

Reply:
We should keep the default commands as is and if you need something like this call it something else 🙂

Reply:
I changed it as default.